src/chip.tsx

```diff
@@ -76,24 +101,59 @@ const Chip: React.FC<ChipProps> = ({Icon, children, id, dataAttributes, active,
                 </Box>
             </Box>
         );
-    } else {
-        const isInteractive = active !== undefined;
+    }
+    const isInteractive = active !== undefined || props.href || props.onPress || props.to;
+
+    const renderContent = () => (
+        <Box
+            className={classnames(
+                styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
+                {
+                    [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
+                }
+            )}
+            paddingLeft={paddingLeft}
+            paddingRight={12}
+            {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+        >
+            {body}
+        </Box>
+    );
+
+    if (props.onPress) {
         return (
-            <Box
-                className={classnames(
-                    styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
-                    {
-                        [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
-                    }
-                )}
-                paddingLeft={paddingLeft}
-                paddingRight={12}
-                {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+            <BaseTouchable trackingEvent={props.trackingEvent} onPress={props.onPress} as="a">
```

Comment:
why `as="a"` ?

Reply:
If I remove the as="a"

The styles change due to BaseTouchable:
![image](https://github.com/Telefonica/mistica-web/assets/1775361/86e15ffa-090a-4484-b2bc-9f54e64b1e54)

Seems like button is styled

Reply:
hmmm, I see. I think I'd apply some styles to reset the button builtin styles, but I prefer this to be a `button`, not an `a`.
Something like this should work:
```js
sprinkles({
    border: 'none',
    background: 'transparent',
    padding: 0,
})
```

Reply:
applied css reset and reverted to button